src/classes/Hacker.js

```diff
@@ -2,7 +2,9 @@ import AiMove from './AiMove'
 import {store} from '../store/store'
 
 export default class Hacker {
-
+  /**
+   * Constructor for this AI personality.
+   */
   constructor() {
```

Comment:
It looks like there may be some common code across the AIs. I'll open a report to look at refactoring these.